reference/pipeline/stages/index.md

```diff
@@ -208,3 +208,11 @@ Scale a Kubernetes object created from a manifest.
 
 ### Undo Rollout (Manifest)
 Rollback a manifest a target number of revisions.
+
+### Wercker
```

Comment:
This should go under the "General" section instead of under the "Kubernetes" section.

Reply:
thanks - done